### PR TITLE
Clarify that profiling is not supported on serverless.

### PR DESCRIPTION
**Pull request by hdafgard**
### What does this PR do?

Adds text clarifying that the profiling product does not support serverless.

### Motivation

Customers not finding out that profiling doesn't support serverless without a lot of hassle.

### Preview

https://docs-staging.datadoghq.com/henrik.dafgard/profiling-serverless/tracing/profiler/enabling/go
https://docs-staging.datadoghq.com/henrik.dafgard/profiling-serverless/tracing/profiler/enabling/java
https://docs-staging.datadoghq.com/henrik.dafgard/profiling-serverless/tracing/profiler/enabling/python
https://docs-staging.datadoghq.com/henrik.dafgard/profiling-serverless/tracing/profiler/enabling/ruby
https://docs-staging.datadoghq.com/henrik.dafgard/profiling-serverless/tracing/profiler/enabling/nodejs
https://docs-staging.datadoghq.com/henrik.dafgard/profiling-serverless/tracing/profiler/enabling/dotnet
https://docs-staging.datadoghq.com/henrik.dafgard/profiling-serverless/tracing/profiler/enabling/php
https://docs-staging.datadoghq.com/henrik.dafgard/profiling-serverless/tracing/profiler/enabling/linux

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/tracing/profiler/enabling/dotnet.md [hunk 1]
```diff
@@ -41,6 +41,8 @@ Datadog .NET Profiler is currently in public beta. Datadog recommends evaluating
 
 Any language that targets the .NET runtime, such as C#, F#, and Visual Basic.
 
+Profiling is not supported on serverless platforms.
```
Comment (felixge) [inline content/en/tracing/profiler/enabling/dotnet.md:44]: NIT:

```suggestion
Profiling is not supported on serverless platforms such as AWS Lambda.
```

Reason: People might type something like "datadog profiling lambda" into google or Ctrl+F on the docs page. Not sure if Lambda is the main platform, but 3/3 Go customers who tried to use Go profiling on serverless where using lambda.

**Review by felixge — APPROVED** [review]
LGTM, see nit.

File: content/en/tracing/profiler/enabling/dotnet.md (hunk 1 above)
Comment (dbenamydd) [reply content/en/tracing/profiler/enabling/dotnet.md:44]: One more nit- maybe s/Profiling/Continuous Profiler/. Kari, your call!

Comment (kayayarai) [reply content/en/tracing/profiler/enabling/dotnet.md]: Yep, I like those suggestions. I also nudged it around on a couple pages. Updated in a commit.

**Review by kayayarai — APPROVED** [review]

**Pull request merged**